experiment_config/bidaf.json

```diff
@@ -1,6 +1,10 @@
 {
   "dataset_reader": {
     "type": "squad",
+    "tokenizer": {
```

Comment:
ugh. This should be the default surely, rather than our hand-rolled one? I need to add this to the SRL model.